packages/expo-task-manager/ios/EXTaskManager/EXTaskService.m

```diff
@@ -76,19 +76,19 @@ - (void)registerTaskWithName:(NSString *)taskName
                      options:(NSDictionary *)options
 {
   Class unversionedConsumerClass = [self _unversionedClassFromClass:consumerClass];
-
+  
```

Comment:
Looks like you have weird formatting settings in Xcode 😅 We'd like to trim whitespaces.